timemachine/fe/rbfe.py

```diff
@@ -24,7 +24,7 @@
 from timemachine.potentials import jax_utils
 
 
-def get_batch_U_fns(bps, lamb):
+def get_batch_U_fns(bps):
     # return a function that takes in coords, boxes, lambda
```

Comment:
nit: delete ref from comment

Reply:
Fixed in bfb226ccf18f4dbbd03de0e287df7bcac708b889